src/components/Home/Hero.astro

```diff
@@ -13,110 +13,111 @@ const blogLists = blogs
   .slice(0, 2);
 ---
 
-<Header>
-  <div
-    id="carouselExampleControls"
-    class="relative"
-    data-te-carousel-init
-    data-te-ride="carousel"
-  >
-    <!--Carousel items-->
+<header
+  class=`text-center relative text-white  p-4 md:p-12 h-full w-full min-h-[40rem]`
+  style=`background-image: linear-gradient(to bottom right, rgba(0,0,0,0.1), rgba(0,0,0,0.1)), url(${heroImage.src}); background-size: cover; background-position: center`
+  data-te-carousel-init
+  data-te-ride="carousel"
+  id="carouselExampleControls"
+>
+  <!--Carousel items-->
+  <div class="h-full mx-auto my-auto">
     <div
-      class="relative w-full overflow-hidden after:clear-both after:block after:content-['']"
+      class="relative w-full mx-auto my-auto overflow-hidden after:clear-both after:block after:content-['']"
     >
       <!--First item-->
       <div
         class="relative float-left -mr-[100%] mb-3 w-full transition-transform duration-[600ms] ease-in-out motion-reduce:transition-none"
         data-te-carousel-item
         data-te-carousel-active
       >
-        <Image height={300} src={logoImage} alt="openAMP Logo" />
+        <div class="w-full h-full flex flex-col justify-center items-center">
+          <Image height={300} src={logoImage} alt="openAMP Logo" />
 
-        <p class="leading-normal text-center max-w-xl text-xl mb-4">
-          {data.main_body}
-        </p>
-        <a
-          onclick="
-          window.open('https://github.com/OpenAMP', '_blank')
-          "
-          class="bg-openampred align-middle mt-2 mx-auto p-4 rounded text-md text-center transition ease-in-out hover:bg-red-600 text-white"
-          >View on Github</a
-        >
+          <p class="leading-normal text-center max-w-xl text-xl mb-4">
+            {data.main_body}
+          </p>
+          <a
+            href="https://github.com/OpenAM"
```

Comment:
Is that missing a P from the GitHub URL?

Reply:
woops, must have happened after I initially pushed 404  fix, as I tested it at that point.